mscore/musescore.cpp

```diff
@@ -4599,6 +4599,26 @@ void MuseScore::cmd(QAction* a)
                QString("Command %1 not valid in current state").arg(cmdn));
             return;
             }
+      if (cmdn == "palette-search") {
+            PaletteBox* pb = getPaletteBox();
+            QLineEdit* sb = pb->searchBox;
+            sb->setFocus();
+            if (pb->noSelection())
```

Comment:
Yes, this won't happen if the search box is clicked, but it's not a problem. If the search box is clicked, PaletteBox::eventFilter will be called, and when up/down/enter is pressed PaletteBox::navigation will be called, which sets keyBoardNavigation to true if there is no selection. So, it works even if the shortcut command for the search box is not used.